test/compilable/test20318.d

```diff
@@ -5,3 +5,9 @@ void main()
 {
     throw new Exception("msg");
 }
+
+void test19998()
+{
+    throw new Exception("");
```

Comment:
I don't get how this addition is any different from the already existing test.

Reply:
It seems I fixed a duplicate of the bug you fixed. I prefer the comment you made, and my fix was slightly more efficient, so I combined the two.